api/external/data_feed/data_feed.proto

```diff
@@ -0,0 +1,136 @@
+syntax="proto3";
+
+package chef.automate.api.datafeed;
+option go_package = "github.com/chef/automate/api/external/data_feed";
+
+import "google/api/annotations.proto";
+
+// for option (chef.automate.api.policy)
+import "components/automate-grpc/protoc-gen-policy/api/annotations.proto";
+// for option (chef.automate.api.iam.policy)
+import "components/automate-grpc/protoc-gen-policy/iam/annotations.proto";
+
+service DatafeedService {
+    rpc AddDestination(Destination) returns (DestinationResponse){
+		option (google.api.http) = {
+			post: "/datafeed/destination"
+			body: "*"
+		};
+		option (chef.automate.api.policy) = {
+			resource: "datafeed:destination"
+			action: "add"
+		};
+		option (chef.automate.api.iam.policy) = {
+			resource: "datafeed:destination"
+			action: "datafeed:destination:add"
+		};
+	};
+    rpc GetDestination(DestinationId) returns (Destination) {
+		option (google.api.http) = {
+			get: "/datafeed/destination/{id}"
+		};
+		option (chef.automate.api.policy) = {
+			resource: "datafeed:destination:{id}"
+			action: "read"
+		};
+		option (chef.automate.api.iam.policy) = {
+			resource: "datafeed:destination:{id}"
+			action: "datafeed:destination:get"
+		};
+	};
+    rpc DeleteDestination(DestinationId) returns (DestinationResponse){
```

Comment:
The RPC should not share the same request parameter `DestinationId`. Create a new one for each RPC. This is because changing the request parameter for an RPC will break existing users, but adding a new filed to a request parameter does not effect existing users. If later the "GetDestination" needs a filters field it can be easily added.